README.md

```diff
@@ -35,8 +23,9 @@ A gradle wrapper is included in the root of this repository and can be called fr
 ### The Example Sub Project:
 1. Copy the example properties file in `example/src/main/resources/` as `application.properties`
 1. Change values for all the appropriate properties. The example file holds sane values for most properties.
-1. Setup Run and Debug configurations. The command line run command is `./gradlew bootRun`
+1. In the command line, run `docker-compose up --build` to create and start all the services from the configuration. 
```

Comment:
same comment as above - for developing/debugging running it through an ide or using gradlew is still the preferred way, just move the docker-compose instructions to a new quick start section
(the only thing needed is copying application.properties, no need to change anything in there since the test profile will override any values already